Header header
string object
string target
float64 x_offset
float64 y_offset
float64 z_offset
bool gripper_open
